tfs/__init__.py

```diff
@@ -15,4 +18,4 @@
 read = read_tfs
 write = write_tfs
 
-__all__ = [read, write, TfsDataFrame, __version__]
+__all__ = [concat, read, write, TfsDataFrame, __version__]
```

Comment:
why only concat? didn't you also add append and merge?

Reply:
These are methods of `TfsDataFrame`